staging/src/k8s.io/apimachinery/pkg/util/runtime/runtime_test.go

```diff
@@ -144,15 +144,20 @@ func captureStderr(f func()) (string, error) {
 	defer func() { os.Stderr = bak }()
 
 	resultCh := make(chan string)
+	errCh := make(chan error)
 	// copy the output in a separate goroutine so printing can't block indefinitely
 	go func() {
 		var buf bytes.Buffer
-		io.Copy(&buf, r)
+		_, copyErr := io.Copy(&buf, r)
```

Comment:
s/copyErr/err, more consistent with the code.

Reply:
I'd stick with `copyErr` to make it clear where the error came from. Also, this is masking the variable:

```suggestion
		_, copyErr = io.Copy(&buf, r)
```

Reply:
Makes sense if the code is changed to use var instead of `errCh` as you pointed out earlier. If not, the error is being sent to the channel in the following line and `err` would be sufficient.

Reply:
Agree with @liggitt . showing error's source is better no matter if it is copied to an outside variable or sent to a channel.